src/lib/y2firewall/clients/installation_finish.rb

```diff
@@ -81,6 +81,9 @@ def configure_firewall
       # Convenience method to enable / disable the firewalld service depending
       # on the proposal settings
       def configure_firewall_service
+        # do not run in autoyast as it is done in second stage (bsc#1177778)
+        return if Yast::Mode.auto
```

Comment:
As commented in #139 it probably should not be done in master as it should be already handled.

Reply:
https://github.com/yast/yast-firewall/blob/a4a998c6e6e215e6f6d0492ae644e70b7567e870/src/lib/y2firewall/clients/installation_finish.rb#L63

Reply:
After checked it is ok to be added.